.github/workflows/verify_newproject_script.yml

```diff
@@ -0,0 +1,37 @@
+name: Verify newproject script
+
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+    branches: [ develop ]
```

Comment:
Just curious, any reason we added `branches: [ develop ]` ? I'm thinking if it would be better to let the run script on all branches 🙏 

Rest LGTM 🚀

Reply:
@Tuubz We only need to trigger when we merge the new updates target to `develop` branch. I am curious why do we need to execute on `main` or `release`? 😄

Reply:
I was thinking about the following case:

1. Create pull request `ONE`, with source branch `A` and target branch `develop`
2. Create pull request `TWO`, with target branch `A`

Then the verify script would not run for pull request `TWO` until we merge pull request `ONE`

Not a big issue, I guess 🙏